src/tests/ftest/object/object_integrity.py

```diff
@@ -11,7 +11,7 @@
 
 from pydaos.raw import (DaosContainer, IORequest, DaosObj, DaosApiError)
 from apricot import TestWithServers
-
+from general_utils import create_string_buffer
```

Comment:
(pylint-syntax-error) Cannot import 'general_utils' due to syntax error 'invalid syntax (<unknown>, line 888)'